### DISCUSSION TITLE
MIMIC Notes

### DISCUSSION THREAD

**Discussion by BassieWitkin** [MIT-LCP/mimic-code#1682] [MIMIC-General]
Hi, 
I am working on creating a model using medical data. I am looking at pretrained models and trying to get an understanding of how they were trained. Most of the models were trained using the MIMIC III dataset. Is there a way to get a demo of the notes, even just a few rows?
Thanks in advance!

**Comment by alistairewj** [comment]
No. The demo intentionally excludes notes. If you would like to access the deidentified free-text clinical notes you will have to apply for access to the database. 
https://mimic.mit.edu/docs/gettingstarted/
